config/config.go

```diff
@@ -45,19 +45,63 @@ func parseFile(c *Config, content []byte) error {
 	return nil
 }
 
-func isValidCheck(check string, checks []string) bool {
-	for _, validCheck := range checks {
-		if strings.EqualFold(check, validCheck) {
-			return true
-		}
-	}
-	return false
+type CheckName = string
+
+// Redefining check names here to avoid circular imports.
+const (
+	BinaryArtifacts      CheckName = "Binary-Artifacts"
+	BranchProtection     CheckName = "Branch-Protection"
+	CIIBestPractices     CheckName = "CII-Best-Practices"
+	CITests              CheckName = "CI-Tests"
+	CodeReview           CheckName = "Code-Review"
+	Contributors         CheckName = "Contributors"
+	DangerousWorkflow    CheckName = "Dangerous-Workflow"
+	DependencyUpdateTool CheckName = "Dependency-Update-Tool"
+	Fuzzing              CheckName = "Fuzzing"
+	License              CheckName = "License"
+	Maintained           CheckName = "Maintained"
+	Packaging            CheckName = "Packaging"
+	PinnedDependencies   CheckName = "Pinned-Dependencies"
+	SAST                 CheckName = "SAST"
+	SBOM                 CheckName = "SBOM"
+	SecurityPolicy       CheckName = "Security-Policy"
+	SignedReleases       CheckName = "Signed-Releases"
+	TokenPermissions     CheckName = "Token-Permissions"
+	Vulnerabilities      CheckName = "Vulnerabilities"
+	Webhooks             CheckName = "Webhooks"
+)
+
+var allValidChecks []string = []string{
+	BinaryArtifacts,
+	BranchProtection,
+	CIIBestPractices,
+	CITests,
+	CodeReview,
+	Contributors,
+	DangerousWorkflow,
+	DependencyUpdateTool,
+	Fuzzing,
+	License,
+	Maintained,
+	Packaging,
+	PinnedDependencies,
+	SAST,
+	SBOM,
+	SecurityPolicy,
+	SignedReleases,
+	TokenPermissions,
+	Vulnerabilities,
+	Webhooks,
+}
+
+func isValidCheck(check string) bool {
+	return slices.Contains(allValidChecks, check)
 }
```

Comment:
ideally this would be a map?

Reply:
How would a map help?

Reply:
Set membership is traditionally done in a map in constant time, instead of a linear search in a slice.

But at this scale (19 checks), it probably doesn't matter.